flows/actions/send_msg.go

```diff
@@ -122,7 +122,16 @@ func (a *SendMsgAction) Execute(run flows.Run, step flows.Step, logModifier flow
 				}
 
 				evaluatedText = translation.Substitute(evaluatedVariables)
-				templating = flows.NewMsgTemplating(a.Templating.Template, evaluatedVariables, translation.Namespace())
+
+				params := map[string][]flows.TemplateParam{}
```

Comment:
Just moved building the params here and pass that to the `flows.NewMsgTemplating` instead of passing a list of variables